tajo-cli/src/main/java/org/apache/tajo/cli/tsql/DefaultTajoCliOutputFormatter.java

```diff
@@ -206,4 +278,30 @@ public static String parseErrorMessage(String message) {
 
     return message;
   }
+
+  private static boolean detectRealTerminal()
```

Comment:
You seem to borrow this code from Presto. If so, you need to add a comment explicitly.
And why did you exclude comments?